src/rez/cli/test.py

```diff
@@ -14,11 +25,17 @@ def setup_parser(parser, completions=False):
     parser.add_argument(
         "--nl", "--no-local", dest="no_local", action="store_true",
         help="don't load local packages")
+    parser.add_argument(
+        "--interactive", dest="interactive", default=None,
+        help="resolves an interactive environment for running the tests.")
+    parser.add_argument(
+        "--variant", dest="variant", default=0, type=int,
+        help="variant number which should be used to set the interactive env. Works only with --interactive option")
     PKG_action = parser.add_argument(
         "--extra-packages", nargs='+', metavar="PKG",
         help="extra packages to add to test environment")
     PKG_action = parser.add_argument(
-        "PKG",
+        "PKG", nargs='?',
```

Comment:
`-h` does not make it clear what it means if `PKG` is not provided. It also is not clear from the help that `PKG` can be a path (ie `.`). I'm tempted to say that we keep it simpler than this - PKG is either provided or not, and if not, then we're in "developer" mode.